Classes/Domain/Model/MetadataObject.php

```diff
@@ -145,6 +145,13 @@ class MetadataObject extends \TYPO3\CMS\Extbase\DomainObject\AbstractEntity impl
      */
     protected $maxInputLength = 0;
 
+    /**
+     * Embargo field option
+     *
+     * @var boolean
+     */
+    protected $embargo;
```

Comment:
Why is embargo a property off a generic MetadataObject?

Reply:
The MetadataObject is the configuration of a form field. With this property you can configure if a field is an "embargo field".